modules/op-release-notes-1-8.adoc

```diff
@@ -533,3 +533,69 @@ securityContext:
+addon-consolecli-ftrb8                   True    
+addon-openshift-67dj2                    True    
+addon-pac-cf7pz                          True    
+addon-pipelines-fvllm                    True    
+addon-triggers-b2wtt                     True    
+addon-versioned-clustertasks-1-8-hqhnw   False   Error
+pipeline-w75ww                           True    
+postpipeline-lrs22                       True    
+prepipeline-ldlhw                        True    
+rhosp-rbac-4dmgb                         True    
+trigger-hfg64                            True    
+validating-mutating-webhoook-28rf7       True 
+----
++
+.Example: Incorrect `TektonConfig` status
+[source,terminal]
+----
+$ oc get tektonconfig config
+NAME     VERSION   READY   REASON
+config   1.8.1     True 
+----
+
+// https://issues.redhat.com/browse/SRVKP-2556
+
+[id="fixed-issues-1-8-1_{context}"]
+=== Fixed issues
+
+* Before this update, the pruner deleted task runs of running pipelines and displayed the following warning: `some tasks were indicated completed without ancestors being done`. With this update, the pruner retains the task runs that are part of running pipelines.
+// https://issues.redhat.com/browse/SRVKP-2419
+
+* Before this update, `pipeline-1.8` was the default channel for installing the {pipelines-title} Operator 1.8.x. With this update, `latest` is the default channel.
+// https://issues.redhat.com/browse/SRVKP-2412
+
+* Before this update, the {pac} controller pods did not have access to certificates exposed by the user. With this update, {pac} can now access routes and Git repositories guarded by a self-signed or a custom certificate.
+// https://issues.redhat.com/browse/SRVKP-2470
+
+* Before this update, the task failed with RBAC errors after upgrading from {pipelines-title} 1.7.2 to 1.8.0. With this update, the tasks run successfully without any RBAC errors.
+// https://issues.redhat.com/browse/SRVKP-2472
+
+* Before this update, using the `tkn` CLI tool you could not remove the task runs and pipline runs containing a `result` object of `array` type. With this update, you can use the `tkn` CLI tool to remove the task runs and pipeline runs containing a `result` object of `array` type.
+// https://issues.redhat.com/browse/SRVKP-2478
+
+* Before this update, if a pipeline specification contained a task with an `ENV_VARS` parameter of `array` type, the pipeline run failed with the following error: `invalid input params for task func-buildpacks: param types don't match the user-specified type: [ENV_VARS]`. With this update, pipeline runs with such pipeline and task specifications do not fail.
+// https://issues.redhat.com/browse/SRVKP-2422
+
+* Before this update, cluster adminstrators could not provide a `config.json` file to the `Buildah` cluster task for accessing a container registry. With this update, cluster adminstrators can provide the `Buildah` cluster task with a `config.json` file using the `dockerconfig` workspace.
```

Comment:
```suggestion
* Before this update, cluster administrators could not provide a `config.json` file to the `Buildah` cluster task for accessing a container registry. With this update, cluster administrators can provide the `Buildah` cluster task with a `config.json` file by using the `dockerconfig` workspace.
```

Reply:
Fair comment - I did not copy this content into an editor to spell-check and vale - lesson learned.